compiler/src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -101,30 +113,28 @@ class LazyVals extends MiniPhase with IdentityDenotTransformer {
     }
   }
 
-
-  /** Append offset fields to companion objects
-    */
+  /** 
+   * Append offset fields to companion objects.
```

Comment:
There's a bunch of spurious changes in this area. Consider reverting them to have a smaller diff and better preserve `git blame`.